src/telemetry/mod.rs

```diff
@@ -19,13 +19,15 @@ mod errno;
 pub mod event;
 mod metrics;
 mod process;
+mod proxy_label;
```

Comment:
I'm not sure if this needs it's own module...

Reply:
What are you suggesting?

I prefer that mod.rs include only types/fns that are core to the module. Otherwise they become dumping grounds for lots of random unrelated stuff.